fix: Allow for better handling for copied instances

Zoinkwiz opened the pull request:
This specifically came up in A Forgettable Tale... where the platforms with chests in the puzzle section exist a few times in the same space.

This meant sometimes the first LocalPoint found was actually the incorrect point. This change is intended to make it so that all LocalPoints can be considered by Steps before actually drawing arrows and highlights.

A note is I feel that the final section of DS2 might possibly be impacted by this, but I can't really predict in what way.

pajlada commented on src/main/java/com/questhelper/steps/DetailedQuestStep.java:
```diff
@@ -374,23 +382,24 @@ public void renderArrow(Graphics2D graphics)
-				return;
-			}
+			List<LocalPoint> localPoints = QuestPerspective.getInstanceLocalPointFromReal(client, worldPoint);
+			if (localPoints.isEmpty()) return;
```
This check seems superfluous, unless it's faster than letting Java just not loop on line 388

pajlada reviewed (commented):
Just some questions, mostly related to nullability.
Have not tried testing this in game, but the changes _seem_ sane to me

Zoinkwiz commented:
Thanks @pajlada as always for the feedback! I've added in a null removal to getInstanceLocalPointFromReal, so it shouldn't contain null values within the returned list, and updated where it's used to no longer check for null nor empty, with the exception of where we're not iterating but rather taking the first element.
